packages/database/.idea/runConfigurations/All_Tests.xml

```diff
@@ -0,0 +1,17 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
What is it for?

Reply:
This is a unit test run configuration for IntelliJ. We have these for Firestore too (https://github.com/firebase/firebase-js-sdk/tree/master/packages/firestore/.idea/runConfigurations)

Reply:
Interesting.  Does anyone use IntelliJ to develop the JS SDK? isn't it a Java IDE?

Reply:
My entire team does. It's really good for TypeScript development and debugging.